marlowe-integration-tests/test/Language/Marlowe/Runtime/Integration/ApplyInputs.hs

```diff
@@ -729,7 +729,7 @@ whenNonEmptySpec = parallel $ describe "Non-Empty When contracts" $ aroundAll se
           emptyMarloweTransactionMetadata
           [NormalInput $ IDeposit address address ada 1_000_000]
     SubmitFailed msg <- expectLeft "Expected a failure" =<< submit' wallet era txBody
-    liftIO $ msg `shouldContain` "MissingRequiredSigners"
+    liftIO $ msg `shouldContain` "MissingVKeyWitnessesUTXOW"
   it "should accept a Role2 deposit from wallet 2" $ runAsIntegration \(ContractCreated _ ContractCreatedInEra{..}) -> do
```

Comment:
`MissingRequiredSigners` doesn't exist anymore (because it is redundant with `MissingVKeyWitnessesUTXOW`) and some tests were expecting this particular exception : 
    - https://github.com/IntersectMBO/cardano-ledger/blob/0e115dd5a38484da1ac2d72559aff8579b887970/eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Rules/Utxow.hs#L118 
    - https://github.com/IntersectMBO/cardano-ledger/issues/3972